ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/OpenStackInfrastructure.py

```diff
@@ -544,8 +622,26 @@ def process(self, device, results, unused):
                 title='{0}.{1}'.format(hypervisor.get('hypervisor_hostname', ''),
                                        hypervisor['id']),
                 hypervisorId=hypervisor['id'],  # 1
-                hypervisor_type=hypervisor_type.get(hypervisor_id, None),
+                hypervisor_type=hypervisor_type.get(hypervisor_id, '').upper(),
                 hypervisor_version=hypervisor_version.get(hypervisor_id, None),
+                host_ip=results['hypervisor_details'].get(hypervisor_id,
```

Comment:
This appears to be the explanation of your other change.  Where did the requirement for all this come from?

I think this hypervisor modeling stuff needs to go into a separate pull request, it's not related to the cinder work and I don't know where the requirement came from.  

It's possible that this may not work in the vmware hypervisor scenario.  It would at least need need to be tested there, and I would rather not hold up this pull request on that.